app/src/js/common/synchronizer.ts

```diff
@@ -3,8 +3,17 @@ import OrderedMap from 'orderedmap'
 import { Dispatch, Middleware } from 'redux'
```

Comment:
Can we pass a `store` with `getState` and `dispatch` to Synchronizer to remove the usage of Session completely here?

Reply:
This is tricky since the store is not created until after the synchronizer connects and reads the state from the backend. I am planning to tackle this in the next PR focused on better initialization.

Reply:
OK. Let's clean it up later.